documentation/CHANGELOG_EXAMPLE.md

```diff
@@ -0,0 +1,474 @@
+# Changelog QualityOnDemand (example according to release process)
+
+## Table of content
```

Comment:
Per your above example, I changed from just using "pre-release" and "release" to using "alpha", "release-candidate", "public" and "patch" in the titles.

The hick is that this may apply to only one of / the "main" API in case of multiple APIs in the same repo, so not fully sure it is the right approach. but it gives a bit more information rather then being "too simple" - however I can revert back.

Another option would be to ONLY use the release number in the title, as peple anyway have to look at the details.

Thoughts ?